ddsketch/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
```

Comment:
I believe we said we wanted to improve the state of the documentation, starting by improving the `description` field of our crates